internal/pkg/agent/cmd/testdata/otel/elastic-agent.yml

```diff
@@ -0,0 +1,301 @@
+###################### Agent Configuration Example #########################
```

Comment:
WDYT about removing all the comments from this file, so this file also becomes more like the `otel.yml` file?  Just thinking that the comments are not adding much value to test fixture files.